src/util.cpp

```diff
@@ -801,11 +801,9 @@ int GetNumCores()
 
 std::string CopyrightHolders(const std::string& strPrefix)
 {
-    std::string strCopyrightHolders = strPrefix + _(COPYRIGHT_HOLDERS);
-    if (strCopyrightHolders.find("%s") != strCopyrightHolders.npos) {
-        strCopyrightHolders = strprintf(strCopyrightHolders, _(COPYRIGHT_HOLDERS_SUBSTITUTION));
-    }
-    if (strCopyrightHolders.find("Bitcoin Core developers") == strCopyrightHolders.npos) {
+    std::string strCopyrightHolders = strPrefix + strprintf(_(COPYRIGHT_HOLDERS), _(COPYRIGHT_HOLDERS_SUBSTITUTION));
+    if (std::string("Bitcoin Core").compare(COPYRIGHT_HOLDERS_SUBSTITUTION) != 0) {
```

Comment:
or just:
```if (std::string("Bitcoin Core") != COPYRIGHT_HOLDERS_SUBSTITUTION)```
No need to use compare() if you don't need to know the relative sorting.